src/table/components/TableBodyWrapper.tsx

```diff
@@ -76,7 +78,7 @@ function TableBodyWrapper({
             const { id, align } = column;
             const cell = row[id] as Cell;
             const CellRenderer = columnRenderers[columnIndex];
-            const handleKeyDown = (evt: React.KeyboardEvent) => {
+            const handleKeyDown = (evt: React.KeyboardEvent, flag: stardust.Flags = flags) => {
```

Comment:
This shouldn't be needed at all. flags is available on the scope, no need to pass it as an argument to this function